### PR TITLE
Update QUEST_LV_0100.tsv

### PR DESCRIPTION
**Pull request by HelpfulKoreanGirl**
UPDATE: Added translations for lines 9588-9655

### CODE REVIEW

**Comment by HelpfulKoreanGirl** [comment]
Translating some of these quest lines without context is kind of hard >.<

@imcgames Is there any way I could get a beta key for the current CBT?

**Comment by ttgmichael** [comment]
@HelpfulKoreanGirl , you mean you're not in the current CBT S:? PM me on Discord

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -9589,12 +9589,12 @@ QUEST_LV_0100_20151016_009584	$Thank you for saving me.{nl}You must also be a Re
...
 QUEST_LV_0100_20151016_009587	{memo X}
-QUEST_LV_0100_20151016_009588	
+QUEST_LV_0100_20151016_009588	Really?{nl}Is this really okay..{nl}
 QUEST_LV_0100_20151016_009589	{memo X}
 QUEST_LV_0100_20151016_009590	{memo X}
-QUEST_LV_0100_20151016_009591	
+QUEST_LV_0100_20151016_009591	Thank you.{nl}You didn't get caught by Leopoldas, did you?
 QUEST_LV_0100_20151016_009592	{memo X}
-QUEST_LV_0100_20151016_009593	
+QUEST_LV_0100_20151016_009593	Editha
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Editha -> Edita 

(Lithuanian name)

**Comment by HelpfulKoreanGirl** [comment]
I am adding additional translated lines in new commits because every time I do a large chunk of translation at once, the website crashes and deletes all my work.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -9629,34 +9629,34 @@ QUEST_LV_0100_20151016_009624	{memo X}
...
 QUEST_LV_0100_20151016_009648	{memo X}
 QUEST_LV_0100_20151016_009649	{memo X}
-QUEST_LV_0100_20151016_009650	
-QUEST_LV_0100_20151016_009651	
-QUEST_LV_0100_20151016_009652	
-QUEST_LV_0100_20151016_009653	
+QUEST_LV_0100_20151016_009650	Are you almost done yet? If you're going to take responsibility, you should get hustlin'.{nl}Don't just linger around here wasting time.
+QUEST_LV_0100_20151016_009651	Who are you?{nl}And what is this rock you've got here?
+QUEST_LV_0100_20151016_009652	What? You destroyed the tombstone?{nl}We worked so hard to erect that tombstone and you destroyed it?
+QUEST_LV_0100_20151016_009653	I don't need your apology.{nl}Since Damiyonas requested it, I'll start working on this, but please don't ever do that again.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Damiyonas -> Damijonas

**Comment by Sunaries** [comment]
:+1: Good job. :)

**Comment by imcgames** [comment]
:+1:

**Pull request merged**